boards/arm/contextualelectronics_abc/contextualelectronics_abc.dts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2020 Analog Life LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "nRF52840 BLE Cell";
+	compatible = "nordic,nrf52840-ble-cell";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
+	compatible = "nordic,nrf-uart";
+	status = "okay";
+	current-speed = <115200>;
+
+	tx-pin = <7>;
+	rx-pin = <6>;
+};
+
+arduino_serial: &uart1 {
+	status = "okay";
+	current-speed = <115200>;
+
+	/* UART Pin info. */
+	rx-pin  = <20>;
+	tx-pin  = <24>;
+	rts-pin = <17>;
+	cts-pin = <16>;
+
+	/* Use hardware flow control. */
+	hw-flow-control;
+
+	/* BG9x description. */
+	quectel_bg9x {
+		compatible = "quectel,bg9x";
+		label      = "quectel-bg9x";
+		status     = "okay";
+
+		mdm-power-gpios = <&gpio1 5  0>;
+		mdm-reset-gpios = <&gpio1 6  0>;
+		mdm-dtr-gpios   = <&gpio0 22 0>;
+	};
+};
+
+arduino_i2c: &i2c0 {
+	compatible = "nordic,nrf-twi";
+	status = "okay";
+	sda-pin = <27>;
+	scl-pin = <26>;
+
+	/* This I2C device is interfaced with the "BQ25895R" charger. */
+};
+
+&spi2 {
+	compatible = "nordic,nrf-spi";
+	status = "okay";
+	sck-pin  = <19>;
+	mosi-pin = <23>;
+	miso-pin = <21>;
+
+	/* This SPI device is interfaced with the SD card. */
+};
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * https://docs.zephyrproject.org/latest/guides/dts/legacy-macros.html#legacy-flash-partitions
```

Comment:
This page is removed. This comment can be removed.

Reply:
Thanks for pointing out, Will remove.